css/style.css

```diff
@@ -140,13 +140,18 @@
   border-radius: 3px;
 }
 
+.book:hover {
+  cursor: pointer;
+}
```

Comment:
Kindly add the css property in the class itself not on hover.

Reply:
kk boss